### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by jmvegh**
# Pull Request Checklist
 * Merge in the latest Develop branch changes to your branch
 * Remove .pyc files from your repository:
   * Linux/Mac
     * find . -name *.pyc -delete
   * Windows
     * del /S *.pyc
 * Run automatic regression and makes sure everything is passing
 * Did you make a new regression test that covers your new code?
 * Did you update your docstrings?
 * Did you update your headers to include your name and date?
 * Do a final compare with the Develop branch to be sure what you're changing

### CODE REVIEW

**Comment by jmvegh** [comment]
allows you to change lift curve slope and drag polar of the airfoil for Propeller.py

File: trunk/SUAVE/Components/Energy/Converters/Propeller.py [hunk 1]
```diff
@@ -264,7 +275,8 @@ def spin(self,conditions):
         #This is an atrocious fit of DAE51 data at RE=50k for Cd
         #There is also RE scaling
         Re      = (W*c)/nu
-        Cdval = (0.108*(Cl*Cl*Cl*Cl)-0.2612*(Cl*Cl*Cl)+0.181*(Cl*Cl)-0.0139*Cl+0.0278)*((50000./Re)**0.2)
+        
+        Cdval = (cd_coeff[0] *(Cl*Cl*Cl*Cl)+cd_coeff[1]*(Cl*Cl*Cl)+cd_coeff[2]*(Cl*Cl)+cd_coeff[3]*Cl+cd_coeff[4])*((re_coeff[0]/Re)**re_coeff[1])   #(0.108*(Cl*Cl*Cl*Cl)-0.2612*(Cl*Cl*Cl)+0.181*(Cl*Cl)-0.0139*Cl+0.0278)*((50000./Re)**0.2)
```
Comment (planes) [inline trunk/SUAVE/Components/Energy/Converters/Propeller.py]: You can removed the commented out code

Comment (jmvegh) [reply trunk/SUAVE/Components/Energy/Converters/Propeller.py]: done

File: trunk/SUAVE/Components/Energy/Converters/Propeller.py [hunk 2]
```diff
@@ -58,10 +59,15 @@ def __defaults__(self):
         self.prop_attributes.twist_distribution  = 0.0
         self.prop_attributes.chord_distribution  = 0.0
         self.prop_attributes.mid_chord_alignment = 0.0
+        self.prop_attributes.lift_curve_slope    = 2.*np.pi    
+        self.prop_attributes.cd_coeff            = [.108, -.2612, .181, -.0139, .0278]  #coefficients for a 4th degree polynomial fit of Cd(Cl) for the airfoil
+        self.prop_attributes.cd_re_scaling_coeff = [50000., .2]  # scaling = (coeff[0]/Re) **coeff[1]
```
Comment (planes) [inline trunk/SUAVE/Components/Energy/Converters/Propeller.py]: I think we should split out the reynolds scaling. The first number is the reference value at which the drag polar was obtained. The second number is an empirical coefficient from the 241 notes for the reynolds effects of drag.

Also maybe we should change the names. Perhaps:

cd_coefficients
drag_reference_reynolds_number
reynolds_scaling_exponent

Thoughts?

Comment (jmvegh) [reply trunk/SUAVE/Components/Energy/Converters/Propeller.py]: Works for me. Done

**Pull request merged**